src/composer.json

```diff
@@ -35,7 +35,7 @@
         "symfony/mime" : "v5.4.13",
         "twig/twig" : "v3.11.1",
         "guzzlehttp/guzzle": "v7.5.0",
-        "phpoffice/phpspreadsheet": "1.19.0",
+        "phpoffice/phpspreadsheet": "1.29.2",
```

Comment:
Just checked the vulnerability status, `1.29.2` and `1.29.3` are also vulnerable.

Better upgrade to latest version `1.29.6`